packages/@jsii/runtime/test/playback.test.ts

```diff
@@ -54,7 +54,7 @@ function createRecords(): string {
       ),
       env: { ...process.env, JSII_RECORD: records, JSII_NOSTACK: '1' },
       stdio: ['inherit', 'pipe', 'pipe'],
-      timeout: 60_000, // 1 minute
+      timeout: 5 * 60_000, // 5 minutes
```

Comment:
```suggestion
      timeout: 300_000, // 5 minutes
```

Reply:
Why? `5 * 60` feels more readable no?

Reply:
(and don't tell me performance)

Reply:
`// 5 minutes` is jolly well enough to explain this is 5 times 60 times one thousand milliseonds...

Reply:
I won't insist, but from a maintenance perspective, if the future me would need to update this from 5 to 7 minutes, it would be easier if I could just change this number from 5 to 7 instead of having to calculate the total value again.